pkgs/by-name/mg/mgitstatus/package.nix

```diff
@@ -0,0 +1,37 @@
+{ fetchFromGitHub, lib, mgitstatus, stdenv, testers }:
```

Comment:
```suggestion
{ fetchFromGitHub
, lib
, mgitstatus
, stdenv
, testers
}:
```

Small nit to make diffing easier in the future

Reply:
Still feels weird to do this for line lengths <80 haha.